py/server/deephaven/_wrapper.py

```diff
@@ -117,6 +119,39 @@ def _lookup_wrapped_class(j_obj: jpy.JType) -> Optional[type]:
     return None
 
 
+def javaify(obj: Union[JObjectWrapper, jpy.JType, Any]) -> Optional[jpy.JType]:
```

Comment:
```suggestion
def javaify(obj: Any) -> Optional[jpy.JType]:
```
Are the other type hints redundant?

Reply:
Yes, they technically are redundant - in Java (esp lacking unions...) I would certainly just say "object", but given that type hints in python are more of "documentation plus", I was trying to be explicit that there are three "categories" of objects that could be passed in, and will be handled separately. In a different universe/language, this might be done with an overloaded method.

It seemed more descriptive to me this way, but I can see this point too, let me know what you'd prefer.